en/docs/quick-start-guides/deploy-your-first-service.md

```diff
@@ -0,0 +1,139 @@
+# Deploy Your First Service
+
+Choreo, an Internal Developer Platform (IDevP), simplifies the deployment, monitoring, and management of your cloud-native services, allowing you to focus on innovation and implementation.
+
+Choreo allows you to easily deploy services you've created in your preferred programming language in just a few steps.
+
+In this guide, you will:
+
+- Use a pre-implemented service that has resources to maintain a book list. 
+- Build and deploy the service in Choreo using the `Nodejs` buildpack. It runs on port 8080.
+- Test the service.
+
+## Prerequisites
+
+1. You will need a GitHub account with a repository that contains your service implementation. To follow this guide you can fork the [Choreo sample book list service repository](https://github.com/wso2/choreo-sample-book-list-service/), which contains the sample for this guide.
+2. The Choreo GitHub App requires the following permissions:
+    - Read access to issues and metadata
+    - Read and write access to code, pull requests, and repository hooks.
+
+### Learn the repository file structure
+
+Let's familiarize ourselves with the key files in the sample greeter application. The below table gives a brief overview of the important files in the greeter service.
+
+!!! note 
+    The following file paths are relative to the path <choreo-sample-book-list-service>/
+    
+|Filepath               |Description                                                                   |
+|-----------------------|------------------------------------------------------------------------------|
+|app.mjs	            |The Node.js (JavaScript) based service code.|
+|.choreo/endpoints.yaml	|Choreo-specific configuration that provides information about how Choreo exposes the service.|
+|openapi.yaml	        |OpenAPI contract of the service. This is required to publish our service as a managed API. This openapi.yaml file is referenced by the .choreo/endpoints.yaml.|
+
+Let's get started!
+
+### Configure the service port with endpoints
+
+We expect to run our book list service on port 8080. To securely expose the service through Choreo, we must provide the port and other required information to Choreo. In Choreo, we expose our services with endpoints. You can read more about endpoints in our [endpoint documentation](https://wso2.com/choreo/docs/choreo-concepts/endpoint/).
```

Comment:
Shall we rephrase to avoid the use of "we".

Reply:
done